ada_2_fhir-r4/zibs2020/ada_instance/nl-core-TreatmentObjective-02.xml

```diff
@@ -0,0 +1,17 @@
+<behandeldoel_registratie app="zib2020-nl" shortName="behandeldoel_registratie" formName="behandeldoel" transactionRef="2.16.840.1.113883.2.4.3.11.60.121.4.2.13.5" transactionEffectiveDate="2020-09-01T00:00:00" versionDate="" prefix="zib2017-" language="nl-NL" title="BehandelDoel-GewensteZorgResultaat" desc="" id="232ea02d-aecd-4881-aea0-2daecd5881fd">
+    <behandeldoel conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.13.5.1">
+        <onderwerp conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.1">
+            <patient-id value="111222333" root="2.16.840.1.113883.2.4.6.3" conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.2"/>
+        </onderwerp>
+        <referenties conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.3">
+            <probleem conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.5.1.1" id="pb-1">
+                <probleem_type conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.5.1.8" value="4" code="409586006" codeSystem="2.16.840.1.113883.6.96" displayName="Klacht"/>
+                <probleem_naam code="267038008" codeSystem="2.16.840.1.113883.6.96" displayName="Oedeem" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.5.1.3"/>

```

Comment:
Use ICD-10 NL code R60 or R60.9 instead, since this SNOMED code is not part of ProbleemNaamCodelijst (see validation error).

Reply:
Changed for a SNOMED code from: 'SNOMED CT: ^11721000146100|Dutch nursing problem simple reference set| -- which should be included in the ProbleemNaamCodelijst.